rudder-web/src/main/scala/com/normation/rudder/web/services/LogDisplayer.scala

```diff
@@ -139,11 +139,7 @@ class LogDisplayer(
           createTechnicalLogsTable("${gridName}",[], "${S.contextPath}",function() {${ajaxRefresh.toJsCmd}}, pickEventLogsInInterval);""") &
         // Load data asynchronously
         JsRaw(
-      s"""
```

Comment:
Is it loading the data directly ? the goal here was to load data only on click on the tab 'technical logs'